VRDR/DeathRecord.md

```diff
@@ -348,15 +348,13 @@
   - [Get_Race_NHOPI_Literals()](#M-VRDR-IJEMortality-Get_Race_NHOPI_Literals 'VRDR.IJEMortality.Get_Race_NHOPI_Literals')
   - [Get_Race_OTHER_Literals()](#M-VRDR-IJEMortality-Get_Race_OTHER_Literals 'VRDR.IJEMortality.Get_Race_OTHER_Literals')
   - [Get_Race_W_Literals()](#M-VRDR-IJEMortality-Get_Race_W_Literals 'VRDR.IJEMortality.Get_Race_W_Literals')
-  - [Get_YNU()](#M-VRDR-IJEMortality-Get_YNU-System-String- 'VRDR.IJEMortality.Get_YNU(System.String)')
```

Comment:
Is this file autogenerated? I manually removed these references since the functions are removed I just wanted to make sure I shouldn't be fully regenerating the documentation in some way.